frontend/packages/noobaa-storage-plugin/integration-tests/tests/obcTest.scenario.ts

```diff
@@ -0,0 +1,81 @@
+import { execSync } from 'child_process';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { CreateOBCHandler } from '../views/obcPage.view';
+
+const obcName = 'test-obc';
+const testDeployment = 'test-deployment';
+const BOUND = 'Bound';
+
+let obcHandler;
+
+const deployment = {
```

Comment:
Deployment isn't noobaa-specific, I think we might need it for other tests as well. Why don't we move it to some place outside of noobaa-storage-plugin?

Reply:
Well this is a deployment that we will use temporarily to test OBC's hence I believe it should be within the plugin. But I will move it to a different folder/file where we can keep such mock data and not mingle it with tests, to keep everything clean,